From a7399955ecc6e3d2f2a6d9c1534c6f839c0a0477 Mon Sep 17 00:00:00 2001
From: Chris Talbot <chris@talbothome.com>
Date: Thu, 22 Feb 2024 20:28:16 -0700
Subject: [PATCH] test-util: remove config.h

config.h has G_DISABLE_ASSERT which throws an error with g_test_init ().
It's best to remove it.
---
 unit/test-util.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/unit/test-util.c b/unit/test-util.c
index 6b00126..5748668 100644
--- a/unit/test-util.c
+++ b/unit/test-util.c
@@ -19,10 +19,6 @@
  *
  */
 
-#ifdef HAVE_CONFIG_H
-#include <config.h>
-#endif
-
 #include <stdio.h>
 #include <fcntl.h>
 #include <unistd.h>
-- 
GitLab
